tools/misc/get_image_metas.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
```

Comment:
Please add a description for this script and add an example for it， you can refer to
https://github.com/open-mmlab/mmdetection/blob/83eea4bec684708171f0b6433636df0873e50dac/tools/analysis_tools/optimize_anchors.py#L2